package.json

```diff
@@ -24,11 +24,12 @@
   "author": "Daniel Durante <me@danieldurante.com>",
   "license": "MIT",
   "dependencies": {
-    "auth-net-request": "2.2.2",
+    "auth-net-request": "SC5/authorize-net-request",
```

Comment:
Hi!

I’ve updated the version to 2.2.3 in our fork and verified that it builds (with a clean node_modules) & passes tests correctly.

-Mikael

> On 25 Aug 2015, at 20:38, Daniel Durante <notifications@github.com> wrote:
> 
> In package.json <https://github.com/durango/authorize-net-cim/pull/14#discussion_r37894824>:
> 
> > @@ -24,11 +24,12 @@
> >    "author": "Daniel Durante <me@danieldurante.com>",
> >    "license": "MIT",
> >    "dependencies": {
> > -    "auth-net-request": "2.2.2",
> > +    "auth-net-request": "SC5/authorize-net-request",
> yup, I believe I just updated it, but we'll still need the request dependency below
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/durango/authorize-net-cim/pull/14/files#r37894824>.
>